src/gen-ts.ts

```diff
@@ -60,12 +63,14 @@ function generatePackage(pkg: Analysis): Promise<string> {
   return Promise.resolve(declarations.join('\n'));
 }
 
-function genElementDeclaration(element: Element, declarations: string[], indent: number = 0) {
+function genElementDeclaration(
+    element: Element, declarations: string[], indent: number = 0) {
   if (!element.className) {
     // TODO: handle elements with tagnae, but no exported class
```

Comment:
console warn (and typo)

Reply:
I'll fix this up later, going to go through all the code next anyway.